package-lock.json

```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "ndb-core",
```

Comment:
Can you please undo these changes.

Reply:
Yes I can. Thanks

Reply:
When I remove the tooltip component from EditLongText and EditText, position of the tooltip changes.
Please have a look on the attached image -

![tooltip-error](https://user-images.githubusercontent.com/83791155/127910527-d7997c90-f565-4b1d-b329-b8eff3f24543.PNG)

Reply:
Yes it looks like the `<ng-template>` takes up to much space. You should add some css rules so all the form elements have the same length and the tooltip icon does not go into the next line.

Reply:
Okay I will do that!